packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -194,15 +195,37 @@ export const VtexCommerce = (
         'items',
         'profile.id,profile.email,profile.firstName,profile.lastName,store.channel,store.countryCode,store.cultureInfo,store.currencyCode,store.currencySymbol'
       )
-
-      return fetchAPI(`${base}/api/sessions?${params.toString()}`, {
+      if (getCookie('vtex_session', ctx.headers.cookie)){
+        // cookie set
+        return fetchAPI(`${base}/api/sessions?${params.toString()}`, {
+          method: 'GET',
+          headers: {
+            'content-type': 'application/json',
+            cookie: ctx.headers.cookie,
+          }
+        })
+       
+    }
+    else {
+        // cookie unset -> create session
+        return fetchAPI(`${base}/api/sessions?${params.toString()}`, {
         method: 'POST',
         headers: {
           'content-type': 'application/json',
           cookie: ctx.headers.cookie,
         },
         body: '{}',
       })
+    }
+    },
+    getsessionorder: (): Promise<Session> => {
```

Comment:
What do you think about camelCase for the sake of keeping the pattern?
```suggestion
    getSessionOrder: (): Promise<Session> => {
```

Reply:
Good idea!! Changed :)